### ISSUE TITLE
Add dependency "libwebkit2gtk-4.1" in AUR script

### ISSUE THREAD

**Issue by bestperson-free** [progzone122/warp-cloudflare-gui-rust#14]
Hello, I collected it in Pamac 
but I still couldn’t start it, what can I do? Thank you
I installed the libwebkit2gtk package, it also did not help, it has version 4.0, not 4.1
```
/usr/bin/warp-cloudflare-gui
/usr/share/applications/warp-cloudflare-gui.desktop
/usr/share/icons/hicolor/256x256/apps/warp-cloudflare-gui.png
/usr/share/icons/hicolor/512x512/apps/warp-cloudflare-gui.png
```
Error
sudo warp-cloudflare-gui 
```
warp-cloudflare-gui: error while loading shared libraries: libwebkit2gtk-4.1.so.0: cannot open shared object file: No such file or directory
```

```
Operating System: Manjaro Linux 
KDE Plasma Version: 6.2.4
KDE Frameworks Version: 6.9.0
Qt Version: 6.8.1
Kernel Version: 6.12.6-1-MANJARO (64-bit)
Graphics Platform: Wayland
Processors: 4 × Intel® Core™ i5-5200U CPU @ 2.20GHz
Memory: 11.1 ГиБ of RAM
Graphics Processor: Mesa Intel® HD Graphics 5500
Manufacturer: ASUSTeK COMPUTER INC.
Product Name: TP300LDB
System Version: 1.0
```

**Comment by bestperson-free** [comment]
I solved everything), installed libwebkit2gtk-4.1, launched it

**Comment by progzone122** [comment]
> I solved everything), installed libwebkit2gtk-4.1, launched it

Most likely, you cleared the cache and that's why everything worked :)

**Comment by progzone122** [comment]
For the future, clear the cache like this:
````shell
pacman clean -k 1 && pamac clean -b
````

**Comment by progzone122** [comment]
I understood what the problem was, thanks :)

**Comment by progzone122** [comment]
Fixed [a21951f7260f3382783c3a3590c6f88f795c9081](https://aur.archlinux.org/cgit/aur.git/commit/?h=warp-gui&id=a21951f7260f3382783c3a3590c6f88f795c9081)
